libafl/src/events/launcher.rs

```diff
@@ -167,8 +169,8 @@ where
 impl<'a, CF, MT, S, SP> Launcher<'a, CF, (), MT, S, SP>
 where
     CF: FnOnce(Option<S>, LlmpRestartingEventManager<(), S, SP>, CoreId) -> Result<(), Error>,
-    MT: Monitor + Clone,
-    S: State + HasExecutions,
+    MT: Monitor + Clone + 'static,
+    S: State + HasExecutions + 'static,
```

Comment:
This looks really bad, state cannot be static?

Reply:
yes these guys do not have to be static.

Reply:
yes, i tested something and forgot to revert, I'll pr with the fix